### ISSUE TITLE
Export file is same as source file

### ISSUE THREAD

**Issue by jauling** [mifi/lossless-cut#1328]
### I have a lot of issues to go through, so in order to make it easier for me to help you, I ask that you please try these things first

- [X] Try with the [newest version from GitHub](https://github.com/mifi/lossless-cut/releases/latest)
- [X] Read all the documentation at https://github.com/mifi/lossless-cut and **Known issues / Troubleshooting**
- [X] Search for your problem under [Issues](https://github.com/mifi/lossless-cut/issues) or [Discussions](https://github.com/mifi/lossless-cut/discussions)
- [X] If you have an issue only with one particular file, [use this form instead](https://github.com/mifi/lossless-cut/issues/new?assignees=&labels=&template=file-issue.yml)

### Operating System

MacOS 12

### Steps to reproduce

I've got a 34 second source file, and I want to export a 16-17 second segment

### Expected behavior

I expect to get an exported 16-17 second segment as usual.

### Actual behavior

The exported file is the same size (not exact) as the source
```
% ls -l Waz\ 10\ Juli\ Spiral*.mp4
-rw-r--r--  1 jchou  660531311  111891227 Jul 11 23:15 Waz 10 Juli Spiral-00.00.00.000-00.00.33.899.mp4
-rw-r--r--@ 1 jchou  660531311  111890569 Jul 11 23:15 Waz 10 Juli Spiral.mp4
```

I've also tried both normal and keyframe cut, as well as adding an extra frame before and after the segment. Same behavior. 

### Share log

```
No error occurred.

{
  "state": {
    "filePath": "/Users/jchou/Movies/Waz 10 Juli Spiral.mp4",
    "fileFormat": "mp4",
    "externalFilesMeta": {},
    "mainStreams": [
      {
        "index": 0,
        "codec_name": "h264",
        "codec_long_name": "H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10",
        "profile": "High",
        "codec_type": "video",
        "codec_tag_string": "avc1",
        "codec_tag": "0x31637661",
        "width": 1080,
        "height": 1920,
        "coded_width": 1080,
        "coded_height": 1920,
        "closed_captions": 0,
        "film_grain": 0,
        "has_b_frames": 2,
        "sample_aspect_ratio": "1:1",
        "display_aspect_ratio": "9:16",
        "pix_fmt": "yuv420p",
        "level": 40,
        "color_range": "tv",
        "color_space": "bt709",
        "color_transfer": "bt709",
        "color_primaries": "bt709",
        "chroma_location": "left",
        "field_order": "progressive",
        "refs": 1,
        "is_avc": "true",
        "nal_length_size": "4",
        "id": "0x1",
        "r_frame_rate": "25/1",
        "avg_frame_rate": "25/1",
        "time_base": "1/25000000",
        "start_pts": 0,
        "start_time": "0.000000",
        "duration_ts": 847000000,
        "duration": "33.880000",
        "bit_rate": "26094410",
        "bits_per_raw_sample": "8",
        "nb_frames": "847",
        "extradata_size": 51,
        "disposition": {
          "default": 1,
          "dub": 0,
          "original": 0,
          "comment": 0,
          "lyrics": 0,
          "karaoke": 0,
          "forced": 0,
          "hearing_impaired": 0,
          "visual_impaired": 0,
          "clean_effects": 0,
          "attached_pic": 0,
          "timed_thumbnails": 0,
          "captions": 0,
          "descriptions": 0,
          "metadata": 0,
          "dependent": 0,
          "still_image": 0
        },
        "tags": {
          "language": "und",
          "handler_name": "VideoHandler",
          "vendor_id": "[0][0][0][0]"
        }
      },
      {
        "index": 1,
        "codec_name": "aac",
        "codec_long_name": "AAC (Advanced Audio Coding)",
        "profile": "LC",
        "codec_type": "audio",
        "codec_tag_string": "mp4a",
        "codec_tag": "0x6134706d",
        "sample_fmt": "fltp",
        "sample_rate": "48000",
        "channels": 2,
        "channel_layout": "stereo",
        "bits_per_sample": 0,
        "id": "0x2",
        "r_frame_rate": "0/0",
        "avg_frame_rate": "0/0",
        "time_base": "1/48000",
        "start_pts": 0,
        "start_time": "0.000000",
        "duration_ts": 1627104,
        "duration": "33.898000",
        "bit_rate": "319341",
        "nb_frames": "1590",
        "extradata_size": 5,
        "disposition": {
          "default": 1,
          "dub": 0,
          "original": 0,
          "comment": 0,
          "lyrics": 0,
          "karaoke": 0,
          "forced": 0,
          "hearing_impaired": 0,
          "visual_impaired": 0,
          "clean_effects": 0,
          "attached_pic": 0,
          "timed_thumbnails": 0,
          "captions": 0,
          "descriptions": 0,
          "metadata": 0,
          "dependent": 0,
          "still_image": 0
        },
        "tags": {
          "language": "und",
          "handler_name": "SoundHandler",
          "vendor_id": "[0][0][0][0]"
        }
      }
    ],
    "copyStreamIdsByFile": {
      "/Users/jchou/Movies/Waz 10 Juli Spiral.mp4": {
        "0": true,
        "1": true
      }
    },
    "cutSegments": [
      {
        "start": 4.28,
        "end": 21.24
      }
    ],
    "mainFileFormatData": {
      "filename": "/Users/jchou/Movies/Waz 10 Juli Spiral.mp4",
      "nb_streams": 2,
      "nb_programs": 0,
      "format_name": "mov,mp4,m4a,3gp,3g2,mj2",
      "format_long_name": "QuickTime / MOV",
      "start_time": "0.000000",
      "duration": "33.899000",
      "size": "111890569",
      "bit_rate": "26405632",
      "probe_score": 100,
      "tags": {
        "major_brand": "isom",
        "minor_version": "512",
        "compatible_brands": "isomiso2avc1mp41",
        "encoder": "Lavf59.16.100"
      }
    },
    "rotation": 360,
    "shortestFlag": false,
    "effectiveExportMode": "sesgments_to_chapters"
  },
  "platform": "darwin",
  "version": "3.46.2"
}
```

**Comment by jauling** [comment]
I will comment that I recently upgraded my MacOS from 12.5.1 to 12.6, which might be related?

**Comment by jauling** [comment]
I just ran it again against a different video clip that I was able to cut fine back in August. Same behavior where the output file is basically the same as the input file. Here are some more screenshots:

![image](https://user-images.githubusercontent.com/5121722/196187429-1da963b6-d612-47cc-b6f6-cdfca74866f1.png)

Export options:
![image](https://user-images.githubusercontent.com/5121722/196187623-9b4683e1-b4c6-4f77-b27b-f2bcc074f6aa.png)

Am I just doing something obviously wrong here?

[YI079401-proj.llc.txt](https://github.com/mifi/lossless-cut/files/9800924/YI079401-proj.llc.txt)

**Comment by jauling** [comment]
it doesn't seem like the 'Fix incorrect duration' did much of anything. I didn't bother to export the file, because the export screen showed me the filename which points to the fact that it'll most likely export the whole thing again.

![image](https://user-images.githubusercontent.com/5121722/196195063-85408258-8b5d-4193-83ee-c8f8f48e8908.png)

**Comment by mifi** [comment]
If possible to share the file with me, or a similar file that reproduces this problem, I can try to find out why this is happening and maybe a workaround for it. you can share with finstaden@gmail.com if private

**Comment by jauling** [comment]
Sure, I'll email you. But I don't think it has anything to do with the source files. I can successfully cut the same files from my Windows machine. I originally was processing everything via SMB mounts on each, but the same happens when I copy the files locally on either machine. I suspect something strange is happening in MacOS 12.6, or something else I might have installed on this machine.

**Comment by mifi** [comment]
Thanks for sending over. I believe the file you sent me is different than the one in your initial post. On my computer (mac 12.5) the file shows up as 18.251 sec long. the project you sent me has 1 segment from 00:00:03.555 to 00:00:11.568. When I export, I get a video that's 8 seconds long, which seems correct. Not sure how to reproduce the problem

**Comment by jauling** [comment]
Are you on Apple silicon? I can't imagine running 12.6 MacOS makes a difference?

**Comment by mifi** [comment]
no, intel. are you?

**Comment by jauling** [comment]
@mifi I'm on Apple silicon.

I seem to have resolved the issue. I uninstalled Losslesscut, I went into `~/Library/Application\ Support/` and deleted the `LosslessCut` folder, then re-installed LosslessCut again.

I guess there was some setting that was messing up my exporting, but I really don't know what. Sorry, I wish I could be more helpful to find the root cause, but at least it seems to be resolved now. My exports aren't the same as my source, that's what matters ;-)
